generated/nifake/nifake_service.h

```diff
@@ -93,12 +93,12 @@ class NiFakeService final : public NiFake::Service {
   void Copy(const std::vector<CustomStruct>& input, google::protobuf::RepeatedPtrField<nifake_grpc::FakeCustomStruct>* output);
   CustomStruct ConvertMessage(const nifake_grpc::FakeCustomStruct& input);
   void Copy(const google::protobuf::RepeatedPtrField<nifake_grpc::FakeCustomStruct>& input, std::vector<CustomStruct>* output);
-  std::map<std::int32_t, float> nifakereal64attributevaluesmapped_input_map_ { {1, 3.5f},{2, 4.5f},{3, 5.5f},{4, 6.5f},{5, 7.5f}, };
```

Comment:
I'm going to start a PR to fix this. It's iterating over a set so the ordering changes, it needs to be sorted or at least ordered.